akka-osgi/build.sbt

```diff
@@ -1,6 +1,7 @@
 import akka.{ AkkaBuild, Dependencies, Formatting, OSGi }
 
 AkkaBuild.defaultSettings
+AkkaBuild.dontPublishSettings
```

Comment:
Wont his will lead to it getting published to your local maven repo if you do a `publishLocal` while dist is about what is put in the dist zip-file?